apps/s_client.c

```diff
@@ -1599,37 +1600,44 @@ int s_client_main(int argc, char **argv)
         goto opthelp;
     }
 #endif
-    if (proxystr != NULL) {
-        int res;
+
+    if (connectstr != NULL) {
+        int res = 0;
         char *tmp_host = host, *tmp_port = port;
-        if (connectstr == NULL) {
-            BIO_printf(bio_err, "%s: -proxy requires use of -connect or target parameter\n", prog);
-            goto opthelp;
-        }
-        res = BIO_parse_hostserv(proxystr, &host, &port, BIO_PARSE_PRIO_HOST);
+            res = BIO_parse_hostserv(connectstr, &host, &port,
```

Comment:
The indentation seems off here. Also please separate declarations from the rest of the code by an empty line.

Reply:
You're right. Fixed.

Reply:
There is meanwhile a tool that can be used to check the coding style (formatting etc.) automatically: `util/check-format.pl`.